cmd/verify/verify.go

```diff
@@ -136,23 +136,24 @@ func main() {
 			if err != nil {
 				log.Error(err)
 			}
+
 			hash := sha256.New()
 			if _, err := io.Copy(hash, c4ghr); err != nil {
 				log.Error(err)
 			}
-
 			//nolint:nestif
-			if message.ReVerify == nil {
+			if message.ReVerify == nil || *message.ReVerify == false {
 				// Mark file as "COMPLETED"
 				if e := db.MarkCompleted(fmt.Sprintf("%x", hash.Sum(nil)), message.FileID); e != nil {
 					// this should really be hadled by the DB retry mechanism
 				} else {
-					// Send message to completed
-					c := Completed{
+					// Send message to verified
+					c := Verified{
 						User:     message.User,
 						Filepath: message.Filepath,
 						DecryptedChecksums: []Checksums{
-							{"SHA256", fmt.Sprintf("%x", hash.Sum(nil))},
+							{"sha256", fmt.Sprintf("%x", hash.Sum(nil))},
+							{"md5", "b60fa2486b121bed8d566bacec987e0d"},
```

Comment:
maybe for testing would be better to generate a random hash smth along the lines of:
```
import (
	"crypto/rand"
	"fmt"
)

func randToken() string {
	b := make([]byte, 16)
	rand.Read(b)
	return fmt.Sprintf("%x", b)
}
```
this will help us avoid any conflicts even though we are using the `sha256`

Reply:
Or perhaps even better a md5 hash. What's the point of the random hash @blankdots ?

Reply:
the point is mostly to avoid possible points of conflict anywhere in the flow (e.g. db, testing messages with cEGA), until we get the proper `md5` from the `crypt4gh` go library. Most likely there will be none but just a safety measure.

Reply:
but was just a suggestion, I already approve as is :slightly_smiling_face:

Reply:
Good, letä's merge then